python/ray/data/datasource/file_based_datasource.py

```diff
@@ -350,6 +350,11 @@ def __init__(
         self._paths, self._file_sizes = meta_provider.expand_paths(
             paths, self._filesystem
         )
+        if self._partition_filter is not None:
+            # Use partition filter to skip files which are not needed.
+            pathToSize = dict(zip(self._paths, self._file_sizes))
```

Comment:
Snake-case rather than camel-case.
```suggestion
            path_to_size = dict(zip(self._paths, self._file_sizes))
```

Reply:
@clarkzinzow - yeah, updated.